applications/liberland.md

```diff
@@ -0,0 +1,250 @@
+
+##### Congress
+
+The Congress is a customized pallet, using delegated voting functionality using the
+underlying Democracy pallet from parity. By utilizing this technology we are creating a true
+representative democracy. Representative democracy is used to _enact_ laws and other 
+regulations, both on-chain (as governance) and o-chain (for the state).  
+In the Assembly elections, a Citizen votes _with their tokens_ , producing a   
+proof-of-stake-like meritocracy in Liberland’s politics. 
+Voters will be allowed to rethink their vote, even mid-term. They can rearrange the  
+LLM allotted to the Congressmen based on their performance.   
+
+##### Executive
+Our Executive is a fork of modern parachain council’s implementations such as Polkadot’s Council, though   
+heavily modified. It is composed of four ministers led by one prime minister. The powers    
+of the Executive are to govern directly on-chain within the limits of Congress’s laws   
+and off-chain as well in the metaverse and later the real-life state.   
+
+
+
+##### Treasury
+One of the standing values that Liberland has is that all taxes are paid voluntarily  
+by the citizens of Liberland. Liberland's chain will have an on-chain treasure, that will be used to fund projects and government budges in a fully transparent way.  
+Treasury, we will handle funding proposals, mainly initiated by the Executive and the Congress.
+The Treasury will be fueled by transaction fees and staking as well as by “real estate fees”, when we are able to launch our Metaverse integration in Milestone 2.
+
+
+
+## Project comparison 
+Several projects such as "bit.country", are trying to be an incubator for the metaverse. We, on the other hand,
+have been here for 7 years now, with a clear vision of being a real state both in the real physical world  
+as well as in the virtual Metaverse. Our chain will manage registration for both the physical land in Liberland as well as the virtual metaverse.
+Liberland's on-chain governance systems stands out from current existing systems. Our system is  
+designed and built with the sole purpose of serving as a nation-state. Allow the users to vote differently based on their role, in accordance to Liberland's governance system. 
+
+#### Liberland's Metaverse  
+We currently have a beta version of our metaverse that has been designed by [Zaha Hadid Architects](https://en.wikipedia.org/wiki/Zaha_Hadid_Architects), developed by [Mytaverse](https://www.mytaverse.com/) and cordinated by [Arch Agenda](https://archagenda.com/).   
+Our metaverse is currently being used for internal meetings and conferences, such as our 7th anniversary, that can be viewed here:
+
+[![IMAGE ALT TEXT](http://img.youtube.com/vi/G_1m8ZwqKMw/0.jpg)](http://www.youtube.com/watch?v=G_1m8ZwqKMw "Liberland MetaVerse")
+
+Our metaverse is not currently public, however, we are more then happy to provide a guided private tour of it, if requested.
+
+
+##### Voting   
```

Comment:
The Voting, _[Land Registry](https://github.com/w3f/Grants-Program/pull/831/files#diff-407622b7a9a07b77406d1bfbe02d81142678a43931b171c74a0ceb0509acba25R96) _and_ [Judiciary](https://github.com/w3f/Grants-Program/pull/831/files#diff-407622b7a9a07b77406d1bfbe02d81142678a43931b171c74a0ceb0509acba25R101)_ sections each describe a specific pallet. In the deliverables you choose different terms for the same concept, e.g. _Public Veto Module_. Please either consolidate these so that the same term is used for the same concept in order to make it easier to understand for everybody involved (see also [Ubiquitous Language](https://martinfowler.com/bliki/UbiquitousLanguage.html)). Or alternatively directly embed these 3 sections into the corresponding deliverables.